modules/Library/library_browse.php

```diff
@@ -137,13 +137,13 @@
     echo '</table>';
 
     //Get current filter values
-    $name = isset($_REQUEST['name']) ? trim($_REQUEST['name']) : null;
-    $producer = isset($_REQUEST['producer']) ? trim($_REQUEST['producer']) : null;
-    $category = isset($_REQUEST['category']) ? trim($_REQUEST['category']) : null;
-    $collection = isset($_REQUEST['collection']) ? trim($_REQUEST['collection']) : null;
-    $everything = isset($_REQUEST['everything']) ? trim($_REQUEST['everything']) : null;
+    $name = trim($_REQUEST['name'] ?? null);
+    $producer = trim($_REQUEST['producer'] ?? null);
+    $category = trim($_REQUEST['category'] ?? null);
+    $collection = trim($_REQUEST['collection'] ?? null);
+    $everything = trim($_REQUEST['everything'] ?? null);
 
-    $gibbonLibraryItemID = isset($_GET['gibbonLibraryItemID']) ? trim($_GET['gibbonLibraryItemID']) : null;
+    $gibbonLibraryItemID = trim($_GET['gibbonLibraryItemID'] ?? null);
 
```

Comment:
Doing `trim()` on null value seems odd. Also that would give you empty string `''` instead of `null` (as the original code does). Please make sure that the code below treat these value the same if they are empty string `''` as if they are `null`.